### PR TITLE
Extend cluster details

### PR DESCRIPTION
**Pull request by orymate**
All fields of the cluster status request (the one using only DB) are added to cluster details request (which calls out to the cluster).

### CODE REVIEW

File: api/cluster.go [hunk 1]
```diff
@@ -372,7 +371,8 @@ func GetClusterDetails(c *gin.Context) {
 	if ok != true {
 		return
 	}
-	log.Info("getting cluster details")
+	log.Debugf("getting cluster details for %v", commonCluster)
```
Comment (sagikazarmark) [inline api/cluster.go]: Use structured logging.

File: api/cluster.go [hunk 2]
```diff
@@ -384,14 +384,24 @@ func GetClusterDetails(c *gin.Context) {
 		return
 	}
 
-	log.Info("Start getting API endpoint")
+	status, err := commonCluster.GetStatus()
+	if err != nil {
+		log.Errorf("Error during getting status: %s", err.Error())
```
Comment (sagikazarmark) [inline api/cluster.go]: Use error handler

File: api/cluster.go [hunk 3]
```diff
@@ -384,14 +384,24 @@ func GetClusterDetails(c *gin.Context) {
...
+		c.JSON(http.StatusBadRequest, pkgCommon.ErrorResponse{
+			Code:    http.StatusBadRequest,
```
Comment (tothandras) [inline api/cluster.go]: `NotFound` might be better here

File: api/cluster.go (hunk 1 above)
Comment (orymate) [reply api/cluster.go]: #1523

**Pull request merged**